charts/spire/charts/tornjak-frontend/templates/service.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: {{ include "tornjak-frontend.namespace" . }}
+  name: {{ include "tornjak-frontend.service" . }}
+  labels:
+    {{- include "tornjak-frontend.labels" . | nindent 4 }}
```

```suggestion
  {{- with .Values.service.annotations }}
  annotations:
    {{- toYaml . | nindent 8 }}
  {{- end }}
  labels:
    {{- include "tornjak-frontend.labels" . | nindent 4 }}
  {{- with 
```

To have same features as other charts.